packages/m/megacmd/package.yml

```diff
@@ -0,0 +1,42 @@
+name       : megacmd
+version    : 1.6.3
+release    : 1
+source     :
+    - https://github.com/meganz/MEGAcmd/archive/refs/tags/1.6.3_Linux.tar.gz : d0378d2a6787f72467eadf27c82145f232b456bc7cd8719abb0ce51e7f719796
+    - https://github.com/meganz/sdk/archive/965e4c7cd518d48c5b2fbf0084a244fdecc44b35.tar.gz : a3e1b96f27bdc60a0b46b50e95731a750d2fa18dcd379e0f6d022e62c6dc653e
```

Comment:
Two things:
- Why point to a specific commit instead of a specific git tag of the sdk?
- You can append `#filename` to the source string in order to rename the file. That makes it easier to do things later in the build since you don't need to update the filename in the commands when the source name changes. IE: `https://github.com/meganz/sdk/archive/965e4c7cd518d48c5b2fbf0084a244fdecc44b35.tar.gz#sdk.tar.gz : a3e1b96f27bdc60a0b46b50e95731a750d2fa18dcd379e0f6d022e62c6dc653e` and then `tar -xf $sources/sdk.tar.gz --strip 1 -C $workdir/sdk/`

Reply:
- because i am not quite sure which version of sdk is this
-  will do that

Reply:
Is using `git|https://github.com/meganz/MEGAcmd.git : 1.6.3_Linux` as source is better for this case? will it pull source file recursively?